docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.7"
+
+# YAML Anchors
+
+x-deployment-env: &deployment-env
+  ENV: ${ENV:-development}
+  SG_ENV: ${SG_ENV:-development}
```

Comment:
Is there any need to set the same value for two env?

Reply:
that's bit of a leftover from where we had both defined. no reason to have backwards compatibility for this environment as we're spinning it up fresh each time.